packages/upstream/vhd-format.0.9.1/opam

```diff
@@ -8,7 +8,7 @@ authors: [ "Dave Scott" "Jon Ludlam" ]
 homepage: "https://github.com/mirage/ocaml-vhd"
 bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
 dev-repo: "git://github.com/mirage/ocaml-vhd"
-build: make
+build: [[make "release"]]
 remove: [[make "uninstall"]]
 depends: [
   "ocamlfind"
```

Comment:
We need to update this `opam` file: since we're pointing to `vhd-format` 0.9.1, we should use the opam file corresponding to that version: https://github.com/mirage/ocaml-vhd/blob/master/vhd-format.opam

Reply:
Well spotted!